Dockerfile

```diff
@@ -72,9 +72,9 @@ COPY --from=docs-builder /opt/builds/build ./docs
 COPY server/index.js ./server/index.js
 COPY server/prometheus.js ./server/prometheus.js
 COPY config ./config
-COPY package.json package-lock.json ./
+COPY package.json yarn.lock ./
 
-RUN npm install --production
+RUN yarn --frozen-lockfile
```

Comment:
great!

Reply:
Now that I look at it. do you not want to add --production anymore and suppress installing devDeps?

Reply:
Good call, not sure why I didn't think to do that. Will update!